src/mqt/yaqs/core/libraries/gate_library.py

```
@@ -709,8 +738,8 @@ class Destroy(BaseGate):
 
     Attributes:
         name: The name of the gate ("destroy").
-        matrix: The 2x2 matrix representation of the gate.
-        interaction: The interaction level (1 for single-qubit gates).
+        matrix: The d×d matrix representation of the gate.
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)